core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -476,13 +476,18 @@ class EMConfig {
         }
 
         when (stoppingCriterion) {
-            StoppingCriterion.TIME -> if (maxActionEvaluations != defaultMaxActionEvaluations) {
+            StoppingCriterion.TIME -> if (maxEvaluations != defaultMaxEvaluations) {
                 throw ConfigProblemException("Changing number of max actions, but stopping criterion is time")
             }
 
             StoppingCriterion.ACTION_EVALUATIONS -> if (maxTimeInSeconds != defaultMaxTimeInSeconds ||
                     maxTime != defaultMaxTime) {
-                throw ConfigProblemException("Changing max time, but stopping criterion is based on fitness evaluations")
+                throw ConfigProblemException("Changing max time, but stopping criterion is based on action evaluations")
+            }
+
+            StoppingCriterion.INDIVIDUAL_EVALUATIONS -> if (maxTimeInSeconds != defaultMaxTimeInSeconds ||
+                maxTime != defaultMaxTime) {
+                throw ConfigProblemException("Changing max time, but stopping criterion is based on individual evaluations")
```

Comment:
to avoid code duplication, this can be merged into a single `->` check, eg `StoppingCriterion.ACTION_EVALUATIONS, StoppingCriterion.INDIVIDUAL_EVALUATIONS -> ` (not 100% sure about Kotlin syntax though...)

Reply:
I made all the changes.